Add 'Dual-boot linux mint cloudready' post

likethemammal opened the pull request.

likethemammal commented on _posts/2016-04-26-cloudready-linux-mint.markdown:
```diff
@@ -0,0 +1,166 @@
+title: Dual-Boot Cloudready and Linux Mint
+tags: linux-mint cloudready dual-boot grub
+username: likethemammal
+---
```
need to add cross post to Razer blade post when it goes live

akenn replied:
Crossposts are used for referencing the same post on a different site. I don't have a problem using it for multiple things like this though.

likethemammal replied:
oh didnt know that. i thought it was for a post referencing another post, nvm then

likethemammal commented on _posts/2016-04-26-cloudready-linux-mint.markdown:
```diff
@@ -0,0 +1,166 @@
+
+For a while I've wanted to setup one of my old machines for her to use. The machine would have to be as easy to use as Chrome OS, but capable of running traditional PC games and audio editing software (she's a musician).
+
+As I detailed in [my last post](), I recently got a Razer Blade gaming laptop setup as my personal machine. That freed up the laptop I had been using, an HP Probook 430 G1, to become hers.
```
Add link to razer blade post once it goes live

mediocregopher commented on _posts/2016-04-26-cloudready-linux-mint.markdown:
```diff
@@ -0,0 +1,166 @@
+
+## Prologue
+
+My girlfriend has been on an HP Chromebook 14 for while now. I was originally the one to recommend it and she's been generally satisfied expect for a few issues. One is that, even though there are many web based alternatives to common software, the Chromebook hardware isn't actually capable of running them for very long. Audio editing, flash gaming, and sometimes even regular usage causes scrolli ...
```
`for a* while now`

mediocregopher replied:
`except* for a few issues`

mediocregopher commented on _posts/2016-04-26-cloudready-linux-mint.markdown:
```diff
@@ -0,0 +1,166 @@
+I've provided diagrams below to explain the difference between this and a traditional dual-boot setup. We're aiming for a traditional dual-boot setup with Cloudready and Linux Mint in parallel. We may even try for the triple-boot!
+
+
+![Linux Mint and Cloudready boot structures](http://i.imgur.com/mK8mdFu.png)
```
you can host the image in this repo, and link to it using the relative path. I did that in one of my previous posts, I think the terrain generation one. Then we're not hotlinking to somewhere else for image hosting,

mediocregopher commented on _posts/2016-04-26-cloudready-linux-mint.markdown:
```diff
@@ -0,0 +1,166 @@
+
+#####Setting up the BIOS
+
+Throughout the setup process you may come across a few recommended changes to the BIOS to get everything working. Might as well do these now. This HP machine has its own HP version of the BIOS, so some things may be specific to HP. I tried to keep things as general as possible, but the [archlinux wiki](https://wiki.archlinux.org/index.php/GRUB/EFI_examples) has detailed info for setting the UEFI  ...
```
Shoutouts to arch linux for having such good docs that other distros don't even need their own

mediocregopher commented on _posts/2016-04-26-cloudready-linux-mint.markdown:
```diff
@@ -0,0 +1,166 @@
+As the picture below depicts, Cloudready has made several partitions beyond just the original Windows partition.
+
+
+![Linux Mint partition menu after Cloudready](http://i.imgur.com/ZFkdpWF.png)
```
more images for this repo

mediocregopher commented:
Made some comments, I think one thing I felt was missing while I read through it was that there wasn't really an explanation for *why* you want to dual-boot chromeOS and linux mint. You explained that on the older laptop chromeOS would get slow, but that doesn't really apply to this new laptop which is the subject of the post. What does having mint on the laptop give you?

likethemammal commented:
This post was about putting Mint and cloudready on *my* old machine, which would be estelle's new one. This one has nothing to do with the new razer blade i got. Was that not clear in the post?

likethemammal commented:
K, @mediocregopher addressed your comments and re-worded the prologue a little to emphasize that dual-booting with Cloudready was to ease the transition for Estelle to this new comp
